doc/pages/administrator/095-advanced-operations.md

```diff
@@ -4,6 +4,14 @@ This section provides guides to perform some operations that have to be perfomed
 could result in data loss or other type of irrecoverable damage. *Always be careful while performing
```

Comment:
I would devote a new line to "*Always be careful while performing these operations*" and add a full stop/exclamation mark at the end of the sentence.